src/ape_ethereum/ecosystem.py

```diff
@@ -318,10 +318,11 @@ def encode_calldata(self, abi: Union[ConstructorABI, MethodABI], *args) -> bytes
         return HexBytes(b"")
 
     def decode_returndata(self, abi: MethodABI, raw_data: bytes) -> Tuple[Any, ...]:
-        output_types = [o.canonical_type for o in abi.outputs]
+        output_types_str = [o.canonical_type for o in abi.outputs]
+        output_types = [parse_type(t.dict()) for t in abi.outputs]
```

Comment:
can use `o` instead of `t` here for consistency, if you prefer

Reply:
consistency is always good.. you can make them all `t` if you prefer.